packages/tools/devtools/devtools-browser-extension/src/devtools/TelemetryLogging.ts

```diff
@@ -75,11 +76,17 @@ export class OneDSLogger implements ITelemetryBaseLogger {
 			httpXHROverride: fetchHttpXHROverride,
 		};
 
+		const instrumentationKey = (process.env.DEVTOOL_TELEMETRY_TOKEN as string) ?? "";
+		console.log(
+			formatDevtoolsScriptMessageForLogging(
+				`Injected telemetry token: ${instrumentationKey}`,
+			),
+		);
```

Comment:
I'm not sure about printing the actual key to the console. If we want to know when it was set, maybe we just log "Fluid Devtools Telemetry token is present" inside the `if` in line 102?

Reply:
Updated